src/layouts/components/Editor/extensions/Image/Image.ts

```diff
@@ -40,6 +40,9 @@ export const IsomerImage = Image.extend({
       height: {
         default: "auto",
       },
+      style: {
+        default: null,
```

Comment:
why do we need to specify the default as null?

Reply:
Thanks for the good catch @harishv7! Yeah, I think we should set it to `100%` or `auto`, I will go with `100%` if you guys are okay with it!